bin/load-runner.js

```diff
@@ -97,6 +108,15 @@ var logger = {
 var outputDir = null;
 var saveOutput = false;
 var writeInProgess = 0;
+var rand;
+
+//initialize random number generator
+const seed = parseInt(args.seed);
+if (seed < 0) {
```

Comment:
If the value of `args.seed` is not a number, then the `parseInt` on L114 will return `NaN`. Any comparison between `NaN` and an actual number will evaluate to false. In this case, that'll mean that you get to L118 and call the function like `lcg(NaN)`. This will cause `lcg` to throw a TypeError:

```
const rand = lcg(NaN);
TypeError: lcg()::invalid input argument. Seed must be a positive integer.
//...
```

In fact, even if the value of is `0`, we'll get to L118 (because zero is not less than zero), and that will also cause the error to be thrown:

```
const rand1 = lcg(0);
TypeError: lcg()::invalid input argument. Seed must be a positive integer.
//...
```

I think we can improve this here, something like this:

``` javascript
const seed =  parseInt(args.seed);
const rand = seed ? lcg(seed) : lcg();
```

And set the type of the `seed` option in `yargs` to be `number`, and not give it a default value -- it will default to `NaN`, which will eval to false here, which will mean it doesn't get passed to `lcg`. `0` will also be falsy so it won't get passed either.

`-1` (or other negative numbers) would be truthy though, so it would get passed and cause the error to be thrown. If you want to also cover that case, you can change it to:

``` javascript
const seed =  parseInt(args.seed);
const rand = seed > 0 ? lcg(seed) : lcg();
```